### PR TITLE
Add org.kde.labplot2

### PR DESCRIPTION
**Pull request by gerlachs**

### CODE REVIEW

File: org.kde.labplot2.json [hunk 1]
```diff
@@ -0,0 +1,144 @@
+{
+    "app-id": "org.kde.labplot2",
+    "runtime": "org.kde.Platform",
+    "runtime-version": "5.15",
+    "sdk": "org.kde.Sdk",
+    "command": "labplot2",
+    "rename-icon": "labplot2",
+    "tags": ["stable"],
+    "desktop-file-name-suffix": " (Stable)",
```
Comment (bilelmoussaoui) [inline org.kde.labplot2.json:9]:
```suggestion
```

File: org.kde.labplot2.json [hunk 2]
```diff
@@ -0,0 +1,144 @@
...
+        {
+            "name": "qtmqtt",
+	    "buildsystem": "qmake",
+	    "builddir": true,
+	    "make-install-args": ["INSTALL_ROOT=/app/qtmqtt"],
+	    "sources": [ { "type": "git", "url": "https://github.com/qt/qtmqtt.git",
+                           "commit": "0da764d8fec459ee28b70f68028ada81ed31abf7", "tag": "v5.15.1" } ]
+	},
+        {
+            "name": "libcerf",
```
Comment (bilelmoussaoui) [inline org.kde.labplot2.json:62]: nit pick: indentation is wrong on almost all the modules names

File: org.kde.labplot2.json [hunk 3]
```diff
@@ -0,0 +1,144 @@
...
+                    "sha256": "5b65a20d5c53494ec8f638267fca4a629836b7ac8dd0ef0266834eab270ed4b3"
+                }
+            ]
+        },
+        {
+            "name": "qtmqtt",
+	    "buildsystem": "qmake",
+	    "builddir": true,
+	    "make-install-args": ["INSTALL_ROOT=/app/qtmqtt"],
+	    "sources": [ { "type": "git", "url": "https://github.com/qt/qtmqtt.git",
```
Comment (bilelmoussaoui) [inline org.kde.labplot2.json:58]: nitpick: indentation

**Comment by bilelmoussaoui** [comment]
bot, build org.kde.labplot2

cc @flathub/kde

File: org.kde.labplot2.json [hunk 4]
```diff
@@ -0,0 +1,144 @@
...
+                    "url": "https://download.kde.org/stable/kuserfeedback/kuserfeedback-1.0.0.tar.xz",
+                    "sha256": "5a2f53ebb4b99a280757ca32bd9b686a7764a726e7e4d8bafee33acbb44b9db7"
+                }
+            ]
+        },
+        {
+            "name": "cantor",
+            "buildsystem": "cmake-ninja",
+            "sources": [ { "type": "git", "url": "https://anongit.kde.org/cantor.git",
+                           "commit": "0c46c2b6b231373f0dce9a46dd6acd90fc01ad33", "tag": "v20.11.80" } ],
```
Comment (tsdgeos) [inline org.kde.labplot2.json:88]: Does it really need the unstable version of cantor?

Also in general i'd prefer if you could use tarballs over git commits/tags

File: org.kde.labplot2.json [hunk 5]
```diff
@@ -0,0 +1,144 @@
...
+                    "type": "archive",
+                    "url": "https://download.kde.org/stable/kuserfeedback/kuserfeedback-1.0.0.tar.xz",
+                    "sha256": "5a2f53ebb4b99a280757ca32bd9b686a7764a726e7e4d8bafee33acbb44b9db7"
+                }
+            ]
+        },
+        {
+            "name": "cantor",
+            "buildsystem": "cmake-ninja",
+            "sources": [ { "type": "git", "url": "https://anongit.kde.org/cantor.git",
```
Comment (cho2) [inline org.kde.labplot2.json:87]:
```
                { 
                    "type": "archive", 
                    "url": "https://download.kde.org/stable/release-service/20.08.3/src/cantor-20.08.3.tar.xz",
                    "sha256": "14aefe60ca2ff7aeaf7af8080dd00e80be8d747faeb9cd5a2df88f0d3af358da" 
                },
```

**Comment by cho2** [comment]
bot, build org.kde.labplot2

**Comment by cho2** [comment]
bot, build org.kde.labplot2

**Comment by gerlachs** [comment]
Looks good to me.

**Pull request closed without merging**